components/Footer/index.js

```diff
@@ -51,17 +51,15 @@ const Footer = () => (
           </NoBulletUl>
         </RightBorderFlex>
         <FlexItem grow={1} basis="350px">
-          <h4 style={{ marginBottom: 0 }}> Hva er itDAGENE?</h4>
           <p>
             itDAGENE er en arbeidslivsmesse hvor studenter blir kjent med
             fremtidige arbeidsgivere. Messen arrangeres av studenter for
             studenter, overskuddet går til studentenes ekskursjon i
             tredjeklasse. itDAGENE arrangeres en gang i året av data- og
             kommunikasjonsteknologi ved NTNU i Trondheim.
           </p>
-          <h4 style={{ marginBottom: 0 }}>Hovedsamarbeidspartner</h4>
-          <p>
-            Årets hovedsamarbeidspartner er{' '}
+          <p style={{ paddingTop: 18 }}>
```

Comment:
Dere kan vel egentlig bare fjerne denne, og legge den inn på nytt når dere slipper årets HSP. :smile:

Reply:
Mottatt! Var mest for å prøve meg litt frem her egentlig hehe